### DISCUSSION TITLE
[Proposal] Apply a Bounty/Reward System for Members Assisting with Answering Questions/Issues

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#7104] [General]
Hello,

As the adoption of the Serenity framework continues to grow daily, we're noticing a significant rise in the number of inquiries and issues from users. Althought Serenity team is doing an excellent job assisting, the sheer volume of questions is becoming challenging to address efficiently. To optimize time and resources, please consider to apply a bounty system to incentivize community members to contribute solutions. This approach would not only alleviate the burden on Serenity team but also provide developers with added motivation to engage and collaborate effectively.

A bounty system provider I know is [](https:///)

**Comment by minhhungit** [comment]
This is also applying for serenity CORE team members, when core team member resolve a question and they can get reward, that is also a way community contribute for you guys development

**Comment by VictorTomaili** [comment]
Thank you for the advice. However, we don't have such a plan.
